service/evaluation/evaluation.go

```diff
@@ -427,12 +387,42 @@ func (svc *Service) ListEvaluationResults(ctx context.Context, req *evaluation.L
 	return
 }
 
+// CreateEvaluationResult is a method implementation of the assessment interface
+func (svc *Service) CreateEvaluationResult(ctx context.Context, req *evaluation.CreateEvaluationResultRequest) (res *evaluation.EvaluationResult, err error) {
+	// Validate request
+	err = service.ValidateRequest(req)
+	if err != nil {
+		return nil, err
+	}
+
+	// We only allow manually created statuses
+	if req.Result.Status != evaluation.EvaluationStatus_EVALUATION_STATUS_COMPLIANT_MANUALLY &&
+		req.Result.Status != evaluation.EvaluationStatus_EVALUATION_STATUS_NOT_COMPLIANT_MANUALLY {
+		return nil, status.Errorf(codes.InvalidArgument, "only manually set statuses are allowed")
+	}
+
+	if req.Result.ValidUntil == nil {
+		return nil, status.Errorf(codes.InvalidArgument, "validity must be set")
+	}
+
+	// Check, if this request has access to the cloud service according to our authorization strategy.
+	if !svc.authz.CheckAccess(ctx, service.AccessCreate, req) {
+		return nil, service.ErrPermissionDenied
+	}
+
+	res = req.Result
+	res.Id = uuid.NewString()
```

Comment:
Not quite sure what you mean here; In case of evidences and assessment results we need to generate the UUID externally because both are created externally and refer to each other.